app/models/legal_aid_application.rb

```diff
@@ -1,4 +1,5 @@
-class LegalAidApplication < ApplicationRecord
+# TODO: Refactor this class to make it smaller
+class LegalAidApplication < ApplicationRecord # rubocop:disable Metrics/ClassLength
```

Comment:
I think we need a `# rubocop:enable Metrics/ClassLength` at the bottom of this class

Reply:
You don't need one if `rubocop:disable` is inline
https://rubocop.readthedocs.io/en/latest/configuration/#disabling-cops-within-source-code (bottom of page)

Reply:
I rebased from `master` and used your code